Mathlib/Tactic/Linter/Lint.lean

```diff
@@ -365,12 +366,23 @@ def longFileLinter : Linter where run := withSetOptionIn fun stx ↦ do
     -- `candidate` is necessarily bigger than `lastLine` and hence bigger than `defValue`
```

Comment:
```
    -- Note that either `lastLine ≤ defValue` and `defValue = linterBound` hold or
    -- `candidate` is necessarily bigger than `lastLine` and hence bigger than `defValue`.

Reply:
Note to self: triple-check this logic; I am not fully convinced any more that these are *all* possible cases.